go.sum

```diff
@@ -159,6 +159,8 @@ github.com/Shopify/sarama v1.30.0/go.mod h1:zujlQQx1kzHsh4jfV1USnptCQrHAEZ2Hk8fT
 github.com/Shopify/toxiproxy v2.1.4+incompatible/go.mod h1:OXgGpZ6Cli1/URJOF1DMxUHB2q5Ap20/P/eIdh4G0pI=
 github.com/Shopify/toxiproxy/v2 v2.1.6-0.20210914104332-15ea381dcdae/go.mod h1:/cvHQkZ1fst0EmZnA5dFtiQdWCNCFYzb+uE2vqVgvx0=
 github.com/VividCortex/gohistogram v1.0.0/go.mod h1:Pf5mBqqDxYaXu3hDrrU+w6nw50o/4+TcAqDqk/vUH7g=
+github.com/Wikia/x v0.0.338-0.20221026080858-acc95efca731 h1:oQF+d//xXj11720C76kCnRFXXBB1PuqSY4rzhwW/OW4=
+github.com/Wikia/x v0.0.338-0.20221026080858-acc95efca731/go.mod h1:FRfCy/kGTj71VVJ8BpA8IX11H6hYqcHaJNyYnbZavjg=
```

Comment:
**question:** I don't known go modules but, looks like a downgrade from `github.com/ory/x v0.0.392`

Reply:
it creates a fake tag (go mod tidy) if you set a branch in the go.mod file. the 337v is probably the highest tag we have in Wikia/x

Reply:
yeah it is https://github.com/WIkia/x/tree/v0.0.337